proteus/TwoPhaseFlow/utils/Parameters.py

```diff
@@ -1641,6 +1641,7 @@ def _initializePhysics(self):
         coeffs.flowModelIndex = V_model
         coeffs.me_model = ME_model
         coeffs.VOFModelIndex = VOF_model
+        coeffs.levelSetModelIndex = LS_model
```

Comment:
This is the only real change right? I think it's fine, although the naming schemes seem to not all be uniform. flowModel and levelSet is camel case, me_model is underscored, and VOFModel is neither.

Reply:
Yes, this is because that is how they are defined in the Coefficients class, see `proteus/mprans/MCorr.py`. This is a recurrent problem across models, those names for storing indices are rather inconsistent (some are `ME_model`, `me_model`, or `modelIndex` for  designing their own index for example..)

Reply:
I see. That's fine then for now.